walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -2,22 +2,32 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Stanley 🐈</title>
+    <title>Laura's Portfolio</title>
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
     <div id="content">
-      <h1>Stanley 🐈</h1>
-      <p>This is Stanley.</p>
-      <img src="images/stanley-1.jpg"/>
+      <h1>Laura's Portfolio</h1>
+      <p>This is Laura, I am a rising junior at MIT. I am currently majoring in Computer Science. 
+	I was born and raised in Venezuela and moved to the US to pursue higher education.</p>
+      <img src="images-me/IMG_0469.JPG"/>
+      <h2>Projects</h2>
+      <ul>
+	<li> Worked at the University of Catalonia on an Autism related project.
```

Comment:
Linting: Remove leading space

Reply:
Hi Luke! Thank you for the comments. What do you mean specifically by linting?

Reply:
Ah, linting is essentially checking for formatting issues. (Issues not necessarily meaning that  the code is bad, just that it isn't formatted in the standard way.) https://en.wikipedia.org/wiki/Lint_(software)
wtf/lint